app/controllers/items/detail.js

```diff
@@ -132,6 +139,29 @@ export default GoodcityController.extend(
         .get("lastObject");
     }),
 
+    infoTabSelected: Ember.computed("tabName", function() {
+      return (
+        ["storage_detail", "storage_content", "info"].indexOf(
+          this.get("tabName")
+        ) > -1
+      );
+    }),
+
+    listTabSelected: Ember.computed("tabName", function() {
+      return ["storage_content", "info"].indexOf(this.get("tabName")) > -1;
+    }),
+
+    storageTypeName: Ember.computed("item", function() {
+      let storageType = this.get("item.storageType");
+      if (storageType) {
```

Comment:
```suggestion
      return storageType && storageType.get("name");
```

I think below suggestion will be a better approach for all scenarios like this in future.

https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining

so basically we can do

```suggestion
      return storageType?.get("name");
```

I think we should add support for optional chaining.

@patrixr , thoughts?

Reply:
I'm all for it, should be easy enough to add this plugin : )